tests/unit_tests/test_dqc_us_0004.py

```diff
@@ -3,18 +3,17 @@
 # See PatentNotice.md for patent infringement notice.
 import unittest
 from datetime import datetime, timedelta
-from src.dqc_us_0004 import _assets_eq_liability_equity, _ASSETS_CONCEPT, _LIABILITIES_CONCEPT, _values_unequal
+from dqc_us_rules import dqc_us_0004
 from mock import Mock, patch
```

Comment:
CR this actually needs to change because the python maintainers hate us. Your import should be now be:
```from unittest.mock import Mock, patch```
Pycharm will well yell at you since that isn't where it is in 2.7, but ignore that.